ophyd/areadetector/base.py

```diff
@@ -240,6 +240,34 @@ def get_asyn_digraph(self):
 
         return G, port_map
 
+    def visualize_asyn_digraph(self, *args, **kwargs):
+        '''This generates a figure showing the current asyn port layout.
+
+        This method generates a plot showing all of the currently enabled
+        Areadetector plugin asyn ports and their relationships. The current
+        ports and relationships are found using self.get_asyn_digraph.
+
+        Parameters
+        ----------
+        *args, **kwargs : networkx.draw_networkx args and kwargs.
+            For the allowed args and kwargs see the `networkx.draw_networkx documentation <https://networkx.github.io/documentation/networkx-1.10/reference/generated/networkx.drawing.nx_pylab.draw_networkx.html>`
```

Comment:
This won't format as a link unless you add a `_` after the closing backtick. See [this section of this Sphinx docs](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links). I sympathize: the external link syntax took me forever to commit to memory.